src/lightning/app/core/constants.py

```diff
@@ -20,6 +20,9 @@
 
 
 def get_lightning_cloud_url() -> str:
+    # detect local development
+    if os.getenv("VSCODE_PROXY_URI", "").startswith("http://localhost:9800"):
+        return "http://localhost:9800"
```

Comment:
Are you sure you want to include the hardcoded port number?

Reply:
Yes, it is fine for now. Thanks for asking :)